tests/components/wake_on_lan/conftest.py

```diff
@@ -0,0 +1,13 @@
+"""Test fixtures for Wake on Lan."""
+from __future__ import annotations
+
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def mock_send_magic_packet() -> AsyncMock:
```

Comment:
Moving this to `conftest` changes the scope of the autoused fixture. Is that on purpose?

Reply:
It doesn't impact other test but I removed autouse so it's called by the relevant tests and thereby no change in use from before